Source/WebInspectorUI/UserInterface/Views/ComputedStyleSection.js

```diff
@@ -129,16 +133,28 @@ WI.ComputedStyleSection = class ComputedStyleSection extends WI.View
         this.needsLayout();
     }
 
+    get variablesGroupType() { return this._variablesGroupType; }
+
+    get isEmpty() {return !this._propertyViews.length; }
+
+    get properties()
+    {
+        if (this._variablesGroupType && this._variablesGroupType !== WI.CSSStyleDeclaration.VariablesGroupType.Ungrouped)
+            return this._style.variablesForType(this._variablesGroupType);
+
+        if (this._style._styleSheetTextRange)
+            return this._style.visibleProperties;
+        else
```

Comment:
Style: the `else` is not necessary because the `if` contains a `return`
```
if (this._style.styleSheetTextRange)
    return this._style.visibleProperties;
return this._style.properties;
```